vagrant/roles/sit.glusterfs/tasks/setup/centos.yml

```diff
@@ -19,15 +17,14 @@
   block:
     - name: Install ansible collections
       become: no
-      shell: ansible-galaxy collection install --no-cache {{ item }}
+      shell: ansible-galaxy collection install {{ item }}
       loop:
         - ansible.posix
         - community.general
         - gluster.gluster
         - ovirt.ovirt
 
     - name: Install ansible collection dependencies
-      pip:
-        name: jmespath
-        executable: pip3.8
+      yum:
+        name: python3.11-jmespath
```

Comment:
Didn't this had issues between centos8 and centos9 ?

Reply:
I thought about it while making the change. Should we split it now or at the time when we could successfully switch to CentOS 9 for GlusterFS cluster?

Reply:
I think it's better to split it now. The playbook should be correct, even if we are not using part of it yet.

Reply:
One thing we could do to avoid too much duplication is to leave the common part in centos.yml, and then include the task centos.yml from both centos8.yml and centos9.yml. After that we can just install the jmespath in the appropriate way for each centos version.

Reply:
Done.